services/data/src/links/RestAPILink/fetchData.ts

```diff
@@ -10,33 +10,66 @@ export const parseContentType = (contentType: string | null) => {
 }
 
 export const parseStatus = async (response: Response) => {
-    if (
+    const accessError =
         response.status === 401 ||
         response.status === 403 ||
+        response.status === 404 ||
```

Comment:
I worry about adding this as an `access` type error, since we lose the ability to differentiate between Not Found and Not Allowed if we lump them together and don't pass through a status code.

Can we leave the `404` check out of this PR and attack that separately?

Reply:
Yeah I agree, I'll remove it 👍